### PR TITLE
This Week in Redox 35.

### PR DESCRIPTION
**Pull request by goyox86**
There we go! 

Sorry for the delay! Been pretty busy.

Give it a good read!

### CODE REVIEW

File: content/news/this-week-in-redox-35.md [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+
+One interesting thing I saw in the kernel was the start of the work on PTI (Page Table Isolation) as a mitigation for the recently discovered [Meltdown vulnerability](https://en.wikipedia.org/wiki/Meltdown_(security_vulnerability)). I was gladly impressed when looking at the [changes](https://github.com/redox-os/kernel/commit/a6550341bbe3614d595dd8ba8113517f3a25f637), very interesting to see how small, localized and the time frame in which they were made. One of the advantages of having a microkernel!
+
+Also in the low level, **syscall**, our system call interface crate, got a bunch of additions like: `WUNTRACED`/`WCONTINUED`, `wif*` functions, and lastly a few more `wait` calls.
+
+A quick look at **Redoxfs** reveals a small change to enter into the null namespace by [@jackpot51](https://github.com/jackpot51).
+
+Moving up to the **drivers** [@ids1024](https://github.com/ids1024) have been busy working on an `ATAPI` driver which gives support for CD-ROM drives. I think is pretty cool!
```
Comment (ids1024) [inline content/news/this-week-in-redox-35.md]: I'm also working in a separate repository on iso9660, the standard filesystem for cdrom drives: https://github.com/ids1024/iso9660-rs

File: content/news/this-week-in-redox-35.md [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+
+# What's new in Redox?
+
+## TL;DR
+
+Hello people! Welcome to the 35th edition of TWiRx!
+
+We started this year with [@jackpot51](https://github.com/jackpot51) being featured on the [Changelog](https://changelog.com/podcast/280) talking about building a secure operating system in Rust and about Redox in general. Make sure you listen to the episode, it's pretty good! (specially the part on which Jeremy says **SPOILER ALERT!**: "Redox is not going away")
+
+This month we've also got the [Redox Crash Challenge](https://www.reddit.com/r/rust/comments/7rv9vw/redox_os_crash_challenge/) going on. An initiative to attempt to cause the system to:
```
Comment (jackpot51) [inline content/news/this-week-in-redox-35.md]: It is worth noting that a number of the issues found have been fixed: 

- An issue was fixed with too many arguments to exec not returning E2BIG: https://github.com/redox-os/kernel/issues/81
- An issue was fixed with the kernel attempting to remap kernel code (and failing due to protections) in: https://github.com/redox-os/kernel/issues/79
- An issue was fixed with address validation allowing overflows: https://github.com/redox-os/kernel/commit/dcb49be48133cf811dcd870e8af5f7ffaffc2ab0
- An issue with `su` allowing login as any user by pressing <kbd>Ctrl</kbd>+<kbd>D</kbd>: https://github.com/redox-os/userutils/commit/02759b4a5a347726e6e81d4ee46a2ade86fd9e1e

File: content/news/this-week-in-redox-35.md [hunk 3]
```diff
@@ -0,0 +1,213 @@
...
+4. [Redocs](http://www.redox-os.org/docs/)
+5. [Fancy GitHub organization](https://github.com/redox-os)
+6. [Our Holy Grail of a Website](http://www.redox-os.org/)
+7. [The Extreme Screenshots](http://www.redox-os.org/screens/)
+
+# New contributors
+ 
+Since the list of contributors are growing too fast, we'll now only list the new contributors. This might change in the future.
+
+Sorted in alphabetical order.
```
Comment (xTibor) [inline content/news/this-week-in-redox-35.md]: This list is not sorted like the text says.

File: content/news/this-week-in-redox-35.md [hunk 4]
```diff
@@ -0,0 +1,213 @@
...
+- [@ids1024](https://github.com/ids1024) AHCI: Refactored duplicated ATA command code into a method. Details [here](https://github.com/redox-os/drivers/pull/23).
+
+## Syscall
+
+Redox Rust Syscall Library
+
+- [@jackpot51](https://github.com/jackpot51) Added `WUNTRACED`. Details [here](https://github.com/redox-os/syscall/commit/88a0512d71c4cfaaa8a9102159416d7f1d09b68a).
+- [@jackpot51](https://github.com/jackpot51) Added `WCONTINUED`. Details [here](https://github.com/redox-os/syscall/commit/8f012900589854cd54ac0a6edb9a3f2b0e017669).
+- [@jD91mZM2](https://github.com/jD91mZM2) Added `wif*` functions. Details [here](https://github.com/redox-os/syscall/pull/22).
+- [@jackpot51](https://github.com/jackpot51) Relased vesrion `0.1.36`. Details [here](https://github.com/redox-os/syscall/commit/7ab7dbe3c80b9b829dd1537fbbacd240092d7019).
```
Comment (leeoniya) [inline content/news/this-week-in-redox-35.md]: "version"

Comment (sajattack) [reply content/news/this-week-in-redox-35.md]: "Relased" should be "Released"

**Comment by goyox86** [comment]
💇 Applied @jackpot51 !

**Comment by jackpot51** [comment]
Looks great!

**Pull request merged**